packages/salesforcedx-vscode-apex-replay-debugger/src/commands/quickLaunch.ts

```diff
@@ -127,8 +121,9 @@ export class QuickLaunch {
 }
 
 export class TestDebuggerExecutor extends LibraryCommandletExecutor<string[]> {
-  protected executionName = nls.localize('debug_test_exec_name');
-  protected logName = 'debug_test_replay_debugger';
+  constructor() {
```

Comment:
To use the `LibraryCommandletExecutor` from the utils package, we need to provide an output channel - created a new channel for the Replay Debugger extension labeled `Apex Replay Debugger`